$OpenBSD: patch-man_feh_pre,v 1.14 2012/05/14 08:20:40 dcoppa Exp $

fix typo

document behaviour of duplicate key bindings
(upstream git commit 9ed58f2c8ca3dd2dc67808676ec437838c4207e6)

document reload_{plus,minus} time delta
(upstream git commit 655169fe6c2b93a58dca133d3144ca674c4e157c)

--- man/feh.pre.orig	Sun Mar 25 13:16:58 2012
+++ man/feh.pre	Mon May 14 10:11:03 2012
@@ -198,7 +198,7 @@ the caption will be looked for in
 Don't display images, print image info according to
 .Ar format
 instead.  See
-.Sx FORMAT SPECIFIEDRS .
+.Sx FORMAT SPECIFIERS .
 .
 .It Cm --cycle-once
 .
@@ -872,7 +872,7 @@ which
 defaults to
 .Pa ~/.config/feh/ .
 If the files are not found in that directory, it will also try
-.Pa /etc/feh/ .
+.Pa ${SYSCONFDIR}/feh/ .
 .Pp
 All config files treat lines starting with a
 .Qq #
@@ -962,6 +962,11 @@ Available modifiers are
 .Ar C No for Control ,
 .Ar S No for Shift and
 .Ar 1 , 4 No for Mod1 and Mod4 .
+To match an uppercase letter like
+.Qq S
+instead of
+.Qq s ,
+the Shift modifier is not required.
 .
 .Pp
 .
@@ -971,6 +976,17 @@ without any keys unbinds it (i.e. the default bindings
 .
 .Pp
 .
+.Em Note :
+Do not use the same keybinding for multiple actions.
+.Nm
+does not check for conflicting bindings, so their behaviour is undefined.
+.
+Either unbind the unwanted action, or bind it to another unused key.
+.
+The order in which you bind / unbind does not matter, though.
+.
+.Pp
+.
 For a list of the
 .Ar action
 names, see
@@ -1123,11 +1139,11 @@ Go backward ~5% of the filelist
 .
 .It + Bq reload_plus
 .
-Increase reload delay
+Increase reload delay by 1 second
 .
 .It - Bq reload_minus
 .
-Decrease reload delay
+Decrease reload delay by 1 second
 .
 .It Ao delete Ac Bq remove
 .
@@ -1247,6 +1263,17 @@ looks like
 .
 .Pp
 .
+.Em Note :
+Do not use the same button for multiple actions.
+.Nm
+does not check for conflicting bindings, so their behaviour is undefined.
+.
+Either unbind the unwanted action, or bind it to another unused button.
+.
+The order in which you bind / unbind does not matter, though.
+.
+.Pp
+.
 For the available modifiers, see
 .Sx KEYS CONFIGURATION SYNTAX .
 .
@@ -1372,7 +1399,7 @@ order it finds them on the hard disk, which is usually
 .
 Show 128x128 pixel thumbnails, limit window width to 1024 pixels.
 .
-.It feh -t -Sfilename -E 128 -y 128 -W 1024 -P -C /usr/share/fonts/truetype/ttf-dejavu/ -e DejaVuSans/8 /opt/images
+.It feh -t -Sfilename -E 128 -y 128 -W 1024 -P -C ${X11BASE}/lib/X11/fonts/TTF/ -e DejaVuSans/8 /opt/images
 .
 Same as above, but enable thumbnail caching in ~/.thumbnails and use a smaller
 font.
